tests/sentry/notifications/utils/test_participants.py

```diff
@@ -331,10 +361,177 @@ def test_fallthrough(self):
 
     def test_without_fallthrough(self):
         ProjectOwnership.objects.get(project_id=self.project.id).update(fallthrough=False)
-        event = self.store_event("no_rule.cpp")
+        event = self.store_event_owners("no_rule.cpp")
 
         assert self.get_send_to_owners(event) == {}
 
+    @with_feature("organizations:streamline-targeting-context")
+    def test_send_to_suspect_committers(self):
+        self.release = self.create_release(project=self.project, version="v12")
+        self.group = self.create_group(
```

Comment:
I do not see `self.group` being associated to `event`. Where does it happen?

I see group id being grabbed from the event itself.
<img width="168" alt="image" src="https://user-images.githubusercontent.com/44410/218189070-711bb27a-2b86-4e77-b695-81fdaab2c9ab.png">

Reply:
Thanks for catching this, `self.group` was not needed here, and has been removed